packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -18,16 +16,104 @@ export interface TypingIndicatorStylesProps extends BaseCustomStylesProps {
   typingString?: IStyle;
 }
 
+/**
+ * Strings of TypingIndicator that can be overridden
+ */
+export interface TypingIndicatorStrings {
+  /**
+   * String template to use when one user is typing. Placeholders: [user].
+   */
```

Comment:
Keep each of these as one large comment block and use `@example` https://tsdoc.org/pages/tags/example/

Reply:
I had originally tried using `@example` but I kept getting this ugly warnings.
![image](https://user-images.githubusercontent.com/79475487/123309239-86d48b00-d4d9-11eb-9343-551d91c4b30f.png)
Would it be okay if I change the rules in eslintrc.js regarding these violations?

Reply:
Does having:
```
@example example desc
`` `
example code
`` `
```
work? (without the space before the last backtick, just put that in so this comment renders correctly)